src/utils/constants/constants.jsx

```diff
@@ -0,0 +1,25 @@
+import markIcon from "src/assets/images/mark.png";
+import penIcon from "src/assets/images/pen.png";
+import binIcon from "src/assets/images/bin.png";
+import logo from "src/assets/images/logo.png";
+import navLogo from "src/assets/images/nav-logo.png";
+import searchIcon from "src/assets/images/search.png";
+import addIcon from "src/assets/images/add.png";
+
+export const ALT_MARK_TEXT = "Mark task as done";
+export const ALT_EDIT_TEXT = "Edit task";
+export const ALT_DELETE_TEXT = "Delete task";
+export const ALT_LOGO_TEXT = "Logo";
+export const ALT_SEARCH_TEXT = "Search";
+export const ALT_NAV_LOGO_TEXT = "Logo";
+export const ALT_ADD_TEXT = "Add Task Button";
+
+export const [
+  ICON_MARK,
+  ICON_PEN,
+  ICON_BIN,
+  LOGO,
+  NAV_LOGO,
+  ICON_SEARCH,
+  ICON_ADD,
+] = [markIcon, penIcon, binIcon, logo, navLogo, searchIcon, addIcon];
```

Comment:
any particular reason for two different type of export? can we follow one convention?